### DISCUSSION TITLE
idea to add "notes" you can put on certain beatmaps

### DISCUSSION THREAD

**Discussion by Terratype** [ppy/osu#22999] [Ideas]
I have bad memory, and I've always wished for a way to add a little "note" to a chart as a quick reminder for the map such as "weird rhythms in this one", "gets really hard at the end", "fun jump map", etc.

I'd find it most helpful in the bottom right of this box here
![image](https://user-images.githubusercontent.com/54338552/227818457-1e8fdc14-096a-4cd0-ad57-17c88b1d71b7.png)
and if there's no note the player is added there could simply nothing there as to not clutter.

Best idea I have as to add one is to add it to the right click context menu for the map
![image](https://user-images.githubusercontent.com/54338552/227818929-84cf7f0d-c68f-4454-8a32-eb5a095dae8f.png)

**Comment by ItsShamed** [comment]
I don't know if it's me, but I feel like this is a little bit of a bloat, a bit too much for a single beatmap. For uses cases like these I use collections, where beatmaps with common (and sometimes esoteric) characteristics can be grouped together, this works quite well.
|  |  |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/39100084/228251705-831d3b1c-20cb-4e7a-8adc-5ca5fa3f633d.png) | ![image](https://user-images.githubusercontent.com/39100084/228258167-3809d196-23ed-4a41-8428-163f5588cb29.png) |

**Reply by Terratype** [reply]
Yeah I see how those can work like that for most players. I had tried using those but they never really had the specificity of what I wanted to remember about the map without having to make tons of collections with 1-2 maps each and then having to scroll across the whole thing seeing if the map I'm about to play was in one of those.
And could you elaborate on what you mean bloat? (like as in bloating file size or visual clutter) I thought my idea of having it only be a context menu item was very nonintrusive as it wouldn't add anything to the screen unless you add a note to that map.

**Reply by TheFearlessDeath500** [reply]
Adding to what ItsShamed said, this does fit better in terms of subcollections, not beatmaps. Also, it’s visual clutter iirc. That’s what they meant by bloat.
But yes, adding some kind of note system to that instead would help— I can see it kind of being used in the beatmap editor though, mostly for mapping purposes. Having some kind of Notepad thing could help, but at the same time it could probably cause some issues.

**Comment by plosee** [comment]
a really good idea imo. i personally would use it to sub-catagorize maps in my collections, since making too many collections is more cluttery in my eyes and i often forget what map has what patterns and it would be really easy if you could 'note' them instead of making a collection for every pattern

**Reply by Rekunan** [reply]
Would having subcollections suffice? #15397 could make for a good use case.

**Reply by plosee** [reply]
this would be a great feature for organizing indeed, preferably i'd be using both though. i feel like a locally stored notes section would compensate for the shortcomings of subcollections like getting too many subcollections and it becoming a pain because a bunch have really specific requirements.
maybe a hide function for collections like windows explorer has with folders aswell would be cool but thats just an improvement that's not for this idea.
